collector/lib/ResourceSelector.h

```diff
@@ -0,0 +1,37 @@
+//
```

Comment:
This file already exists in another PR. Some changes were made to it in this PR.